### PR TITLE
Add script to fix faulty values of bib encodingLevel

### PR DESCRIPTION
**Pull request by sarwik**

### CODE REVIEW

File: whelktool/scripts/cleanups/2019/10/lxl-2594-fix-faulty-encodingLevels/set-correct-bib-encodingLevel-value.groovy [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                             "marc:CoreLevel",
+                             "marc:FullLevel",
+                             "marc:FullLevelMaterialNotExamined",
+                             "marc:LessThanFullLevelMaterialNotExamined",
+                             "marc:MinimalLevel",
+                             "marc:PartialPreliminaryLevel",
+                             "marc:PrepublicationLevel"]
+
+selectBySqlWhere("""
+        collection = 'bib' AND data#>>'{@graph,0,encodingLevel}' IS NOT NULL
```
Comment (niklasl) [inline whelktool/scripts/cleanups/2019/10/lxl-2594-fix-faulty-encodingLevels/set-correct-bib-encodingLevel-value.groovy]: If this is too slow on all bib:s you may gain speed by using a more specific select.

File: whelktool/scripts/cleanups/2019/10/lxl-2594-fix-faulty-encodingLevels/set-correct-bib-encodingLevel-value.groovy [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    """) { data ->
+
+    def (record, instance, work) = data.graph
+
+    if (!VALID_BIB_ENCODING_LEVELS.contains(record['encodingLevel'])) {
+        record.put('encodingLevel', VALID_BIB_ENCODING_LEVELS[0])
```
Comment (niklasl) [inline whelktool/scripts/cleanups/2019/10/lxl-2594-fix-faulty-encodingLevels/set-correct-bib-encodingLevel-value.groovy]: Could extract first value as an `ABBR_LEVEL` constant, use that in the list and use `as Set` on that (for clarity and an admittedly negligible speedup).

**Comment by klngwll** [comment]
> Has AbbreviatedLevel been agreed upon? (That get's overwritten IIRC? Some in the forum discussion thought otherwise and the issue is unclear on what the resolution has been. In the chat MinimalLevel was recommended unless I've mixed up issues?)

Moving from a numerical code to the EncLevelTypes has been a source of confusion from the beginning since they don't correspond to the swedish usage, hence when we speak of "miniminivå" it is actually AbbreviatedLevel:   

```
:AbbreviatedLevel a :EncLevelType ;
    skos:notation "3" ;
    skos:prefLabel "Abbreviated level"@en, "Miniminivå"@sv .

:MinimalLevel a :EncLevelType ;
    skos:notation "7" ;
    skos:prefLabel "Minimal level"@en, "Biblioteksnivå"@sv .
```

File: whelktool/scripts/cleanups/2019/10/lxl-2594-fix-faulty-encodingLevels/set-correct-bib-encodingLevel-value.groovy (hunk 1 above)
Comment (sarwik) [reply whelktool/scripts/cleanups/2019/10/lxl-2594-fix-faulty-encodingLevels/set-correct-bib-encodingLevel-value.groovy]: Good point. I have adjusted the sql query to only select records with encoding Levels set to hold and auth levels.

**Pull request merged**